src/components/api-reference/schemas/measurements.js

```diff
@@ -103,6 +109,7 @@ class Measurements extends React.PureComponent {
           </Tabs>
         </div>
         <DataModelTable fields={FIELDS} />
+
```

Comment:
added some blank lines to separate out sections to improve code readability, let me know if that's a style no-no